net/csshnpd/files/csshnpd.init

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2007-2011 OpenWrt.org
+
+USE_PROCD=1
+START=80
+
+start_instance() {
+	local section="$1"
+
+	config_get_bool enabled     "$section" enabled 1
```

Comment:
Please only one whitespace between so we have:
`config_get_bool enabled "$section" enabled 1`
instead of
`config_get_bool enabled     "$section" enabled 1`

Reply:
Fixed in https://github.com/openwrt/packages/pull/26518/commits/2b080148ae6833ed7d8ae8c5dd14b05b272ecf12